app/views/layouts/blacklight/base.html.erb

```diff
@@ -22,7 +22,7 @@
     <%= content_for(:head) %>
 
     <%# Needed to enable Google Search Console %>
-    <meta name="google-site-verification" content="JuhY7qlmlvIHFXEt0p3Y0yjW617hqz125HaEzj0uWW0" />
+    <meta name="google-site-verification" content="gSKpv970Qyib2XnLvLpjn0IhFmkeRhI1qNYbQNaQGHo" />
```

Comment:
How do we manage these magic numbers?

Reply:
@botimer this was inherited from dul-arclight, but there are [multiple methods available]( https://support.google.com/webmasters/answer/9008080#meta_tag_verification&zippy=%2Chtml-tag:~:text=Choosing%20a%20verification%20method) --- de-coupling this from the application release would probably be preferable.

Reply:
Okay, so something from env or file eventually. No problem for this PR.